docs/pom.md

```diff
@@ -0,0 +1,30 @@
+The following metadata fields can be extracted from a pom.xml file
+
+| Metadata category       | Metadata Field                 | Value    |
```

Comment:
It's SOMEF metadata category and somef metadata field. The `value` is the POM.XML value